packages/types/src/deneb/sszTypes.ts

```diff
@@ -154,6 +156,57 @@ export const BlobSidecar = new ContainerType(
 
 export const BlobSidecars = new ListCompositeType(BlobSidecar, MAX_BLOBS_PER_BLOCK);
 
+export const SignedBlobSidecar = new ContainerType(
+  {
+    message: BlobSidecar,
+    signature: BLSSignature,
+  },
+  {typeName: "SignedBlobSidecar", jsonCase: "eth2"}
+);
+export const SignedBlobSidecars = new ListCompositeType(SignedBlobSidecar, MAX_BLOBS_PER_BLOCK);
+
+export const BlockContents = new ContainerType(
+  {
+    block: BeaconBlock,
+    blobSidecars: BlobSidecars,
+  },
+  {typeName: "BlockContents", jsonCase: "eth2"}
+);
+
+export const SignedBlockContents = new ContainerType(
+  {
+    signedBlock: SignedBeaconBlock,
+    signedBlobSidecars: SignedBlobSidecars,
+  },
+  {typeName: "SignedBlockContents", jsonCase: "eth2"}
+);
```

Comment:
Is this a spec'ed type? Types in `packages/types` should exclusively include spec'ed types. Else define in beacon-node package

Reply:
i think its just beacon-apis serialization type, might be useful in eventually supporting ssz encoded/decoded responses in api, but for now will move into beacon-api package itself and import/use from there :+1:

Reply:
@dapplion  have done some refactoring on this, let me know if you find it suitable now